Dockerfile

```diff
@@ -0,0 +1,13 @@
+from python:3.11-slim
+
+workdir /app
+
+run pwd
+run ls -la /
+
+copy . ./
+run pip install -r requirements.txt
+
+copy . .
```

Comment:
Not sure what line exactly you are talking about as there are 3 in the snippet, so ill just comment on all of them. 

- `copy . ./` is copying all the files, however it should only be copying 'requirements.txt' but I changed it while trying to fix it not copying over, causing issues. It has been replaced to just copy the requirements file now. 
- `run pip...` is installing all the needed packages.
- `copy . .` copies all the project files to the docker container to actually run. 
- Next, it now changes the working directory to '/app/Xena/ so that the relative path of 'google_credentials.json' as set in 'main.py' is correct and doesn't error out.